jbpm-process-svg/src/main/java/org/jbpm/process/svg/SVGImageProcessor.java

```diff
@@ -92,6 +94,12 @@ public static String transform(InputStream svg, List<String> completed, List<Str
     public static String transform(InputStream svg, List<String> completed, List<String> active,
                                    Map<String, String> subProcessLinks, String completedNodeColor,
                                    String completedNodeBorderColor, String activeNodeBorderColor, Map<String, Long> badges) {
+        return transform(svg, completed, active, Collections.emptyList(), subProcessLinks, completedNodeColor, completedNodeBorderColor, activeNodeBorderColor, badges, "");
+    }
+
+    public static String transform(InputStream svg, List<String> completed, List<String> active, List<String> error,
```

Comment:
The updated method signature is not perfect. I'd suggest moving errorNodeBorderColor near other node/border color arguments. Plus, does an additional method make sense? I would say we should change signature of an existing method, cause as long as we provide any border/node color argument we might want to provide the newly introduced `errorNodeBorderColor` too.